src/Domain/Model/BookingDataTransformers/PayPalBookingTransformer.php

```diff
@@ -37,6 +37,7 @@ class PayPalBookingTransformer {
 		'paypal_mc_fee' => 'mc_fee',
 		'paypal_settle_amount' => 'settle_amount',
 		'ext_payment_id' => 'txn_id',
+		'ext_transaction_type' => 'txn_type',
```

Comment:
This will store additional and redundant data in the data blob and it will never be read again. That's why I suggested deriving `ext_payment_status` in PayPalBookingTransformer.

The `ext_` fields were meant as shared payment attributes (credit card also writes some of them). Currently, the FOC reads the ones it knows about when showing the detail view and also exports some of them (see https://github.com/wmde/fundraising-backend/blob/master/src/DataAccess/DonationExport/OutputFieldMapper.php#L89-L94 )